app/src/main/res/xml/preferences.xml

```diff
@@ -37,6 +37,13 @@
             android:title="@string/pref_location_background_title"
             android:summary="@string/pref_location_background_summary"/>
     </PreferenceCategory>
+    <PreferenceCategory
+        android:title="@string/other_settings">
+        <SwitchPreference
```

Comment:
I saw the icon added in source but you aren't referencing it here?

Reply:
Fullscreen icon reference added